charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.31.0
+version: 9.31.1
```

Comment:
Adding a feature, so please make this a minor version bump

Reply:
Done